erpnext/support/doctype/issue/issue.json

```diff
@@ -114,7 +120,7 @@
    "no_copy": 1,
    "oldfieldname": "status",
    "oldfieldtype": "Select",
-   "options": "Open\nReplied\nHold\nClosed",
+   "options": "Open\nReplied\nResolved\nClosed",
```

Comment:
- How is `Resolved` and `Closed` different from each other ??
- What is the user has to put an issue on Hold due to any reason ?? Would that break workflows or any customisations if any ?

Reply:
**Resolved:** When users are reasonably sure that they have provided the customer with a solution to their problem, they can change the status of the issue to Resolved.

**Closed:** The Issue can be set as Closed if the customer acknowledges that the problem has been resolved.

These were introduced to have better clarity about the status of the issue.

> * What is the user has to put an issue on Hold due to any reason ?? Would that break workflows or any customisations if any ?

This is a matter of concern. Should we simulate "Hold" time on Replied as well as the Hold status and not remove Hold?